Makefile

```diff
@@ -17,6 +17,7 @@ RM := rm -f
 CP := cp
 MAKE := make
 DOCKER_TOOLS = docker run -v $$(pwd):/build lndmon-tools
+DOCKER_LINT = docker run -t --rm -v $$(pwd):/app -w /app golangci/golangci-lint:v1.55.2 golangci-lint run -v
```

Comment:
Sorry you lost time on this... The `-tags=tools` in the `tools/Dockerfile` (Line 18) was an oversight on my part in the first place, we removed that in most other projects since it caused this exact problem...
Can you please just remove that too and revert the changes in the `Makefile`?

Reply:
Awesome, that does the trick! Updated.